aspnet/security/anti-request-forgery.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Preventing Cross-Site Request Forgery (XSRF/CSRF) Attacks in ASP.NET Core
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 10/14/2016
+ms.topic: article
+ms.assetid: 09607e8b-a977-4807-81aa-7a5d73bfbacd
+ms.prod: aspnet-core
+uid: security/anti-request-forgery
+---
+
+  # Preventing Cross-Site Request Forgery (XSRF/CSRF) Attacks in ASP.NET Core
+
+[Fiyaz Hasan](https://twitter.com/FiyazBinHasan), David Paquette
+
+See this [word document](https://www.dropbox.com/sh/jfmncpp0z79m9mt/AAAAB98zC4J7f9g_l6egU8nta?dl=0) and [Evolving ASP.NET Apps–Cookie Authentication](https://blogs.msdn.microsoft.com/cdndevs/2015/02/18/evolving-asp-net-appscookie-authentication/)
+
+David Paquette gave me permission to copy his blog if we add him to the authors list.
+
+Cross-site request forgery (also known as XSRF or CSRF) is an attack against web-hosted applications whereby a malicious web site can influence the interaction between a client browser and a web site trusted by that browser. These attacks are made possible because web browsers send authentication tokens automatically with every request to a web site.
+
+Here is an example of a CSRF attack:
+
+1. A user logs into `www.example.com`, using forms authentication.
+
+2. The server authenticates the user and issues a  response that includes an authentication cookie.
+
+3. The attacker uses social engineering to trick the user into clicking on a link to a malacious site.
```

Comment:
I'd rather "The user visits a malicious site". Saying click a link reads like it needs to be a click through. It doesn't.